test/globalize/cache_key_test.rb

```diff
@@ -1,14 +1,16 @@
 # encoding: utf-8
-
 require File.expand_path('../../test_helper', __FILE__)
 
 class CacheKeyTest < MiniTest::Spec
-
   describe '#cache_key' do
     it "changes when the translation is updated" do
       product = with_locale(:en) { Product.create(:name => 'first') }
       original_cache_key = product.cache_key
 
+      # If the test is too fast the new cache_key (based on the `updated_at`
+      # column) will be equal, causing a failing test.
+      sleep(1)
```

Comment:
What about to use [travel_to](http://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel_to)?